components/common/hocs/cloud/with_get_cloud_subcription.tsx

```diff
@@ -5,26 +5,37 @@ import React, {ComponentType} from 'react';
 
 import {isEmpty} from 'lodash';
 
-import {Subscription} from 'mattermost-redux/types/cloud';
+import {Subscription, SubscriptionStats} from 'mattermost-redux/types/cloud';
 
 interface Actions {
-    getCloudSubscription: () => void;
+    getCloudSubscription?: () => void;
+    getSubscriptionStats?: () => void;
 }
 
 interface UsedHocProps {
-    subscription: Subscription;
+    subscription?: Subscription;
+    subscriptionStats?: SubscriptionStats;
     isCloud: boolean;
     actions: Actions;
-    userIsAdmin: boolean;
+    userIsAdmin?: boolean;
```

Comment:
Does optional make sense here for a bool? I would imagine it always defaults to false